src/expressions/operators/compares/Compare.ts

```diff
@@ -46,6 +50,21 @@ class Compare extends Expression {
 			case 'greaterThanOrEqualOp':
 			case 'greaterThanOp':
 				this._compare = 'generalCompare';
+				if (
+					firstType &&
+					secondType &&
+					firstType.type !== ValueType.ITEM &&
+					secondType.type !== ValueType.ITEM &&
+					firstType.type !== ValueType.ARRAY &&
+					secondType.type !== ValueType.ARRAY
```

Comment:
What's gonna happen to array btw?

Reply:
It just isn't going to react to the shortcut

Reply:
THis is done because array is anotated with just ARRAY not the types it contains

Reply:
Oh okay I see thanks